modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,612 @@
+examplecluster-control-plane-0    Running                          3h11m   openshift-control-plane-0   baremetalhost:///openshift-machine-api/openshift-control-plane-0/da1ebe11-3ff2-41c5-b099-0aa41222964e   externally provisioned
+examplecluster-control-plane-1    Running                          3h11m   openshift-control-plane-1   baremetalhost:///openshift-machine-api/openshift-control-plane-1/d9f9acbc-329c-475e-8d81-03b20280a3e1   externally provisioned
+examplecluster-compute-0          Running                          165m    openshift-compute-0         baremetalhost:///openshift-machine-api/openshift-compute-0/3d685b81-7410-4bb3-80ec-13a31858241f         provisioned
+examplecluster-compute-1          Running                          165m    openshift-compute-1         baremetalhost:///openshift-machine-api/openshift-compute-1/0fdae6eb-2066-4241-91dc-e7ea72ab13b9         provisioned
+----
+
+. Remove the old `BareMetalHost` object:
++
+[source,terminal]
+----
+$ oc delete bmh openshift-control-plane-2 -n openshift-machine-api
+
+baremetalhost.metal3.io "openshift-control-plane-2" deleted
+----
++
+After you remove the `BareMetalHost` and `Machine` objects, then the `Machine` controller automatically deletes the `Node` object.
++
+If, after deletion of bmh, the machine node requires excessive time for deletion, the machine node can be deleted using: 
++
+[source,terminal]
+----
+$ oc delete node openshift-control-plane-2
+
+node "openshift-control-plane-2" deleted
+----
++
+Verify the node has been deleted:
++
+[source,terminal]
+----
+$ oc get nodes
+
+NAME                     STATUS ROLES   AGE   VERSION
+openshift-control-plane-0 Ready master 3h24m v1.24.0+9546431
+openshift-control-plane-1 Ready master 3h24m v1.24.0+9546431
+openshift-compute-0       Ready worker 176m v1.24.0+9546431
+openshift-compute-1       Ready worker 176m v1.24.0+9546431
+----
+
+. Create the new `BareMetalHost` object and the secret to store the BMC credentials:
+
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-control-plane-2-bmc-secret
+  namespace: openshift-machine-api
+data:
+  password: <password>
+  username: <username>
+type: Opaque
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-control-plane-2
+  namespace: openshift-machine-api
+spec:
+  automatedCleaningMode: disabled
+  bmc:
+    address: redfish://10.46.61.18:443/redfish/v1/Systems/1
+    credentialsName: openshift-control-plane-2-bmc-secret
+    disableCertificateVerification: true
+  bootMACAddress: 48:df:37:b0:8a:a0
+  bootMode: UEFI
+  externallyProvisioned: false
+  online: true
+  rootDeviceHints:
+    deviceName: /dev/sda
+  userData:
+    name: master-user-data-managed
+    namespace: openshift-machine-api
+EOF
+----
++
+Note: username and password can be found from the other bmh's secrets. The protocol to use in bmc:address: can be taken from other bmh objects.
```

Comment:
Is "bmh" a commonly understood acronym? If not, then suggestion to expand it out to bare metal host:
```suggestion
[NOTE]
====
The username and password can be found from the other bare metal host's secrets. The protocol to use in the `bmc.address` field can be taken from other Bare Metal Host objects.
====
```

Reply:
It is, but it's a specific modifier used in the command itself. Pulling that term out into the descriptions may affect readability to the point that I think I should go with your changes.  Also, SERIOUSLY agreed with you about changing the Note: to [NOTE]. I was afraid of repetition here but I think the change is a big improvement in consistency.